frontend/package.json

```diff
@@ -32,7 +32,6 @@
 		"firebase": "^9.19.1",
 		"framer-motion": "^6.3.15",
 		"graphql": "^15.7.2",
-		"highlight.io": "workspace:*",
```

Comment:
unrelated, but helps with frontend dev time by avoiding to have to run `dev` on the highlight.io workspace to be able to run the frontend